### ISSUE TITLE
Removing parce.li

### ISSUE THREAD

**Issue by 4ndr01d3** [biojs/slush-biojs#16]
Hey there,
I am updating this to avoid the use of the parce.li server by simply excluding the "style" tag in the package json. I was actually thinking to remove the build directory from the .gitignore file, so the bundled css gets into github and can be used in the routes of the sniper in the registry.
Does anyone think this approach is wrong somehow? Is there any reason to keep the build directory out of the repo?
@greenify @emepyc If you are OK with it, can you please add me as a collaborator for the npm module, so I can publish the changes.
Cheers,
Gustavo

**Comment by emepyc** [comment]
I have mixed feelings with pushing the build/ dir.
Does it makes sense if the registry builds the package? optionally providing a cdn url for the sniper in the package.json?

**Comment by greenify** [comment]
I agree with Miguel that committing the "build" directory to git feels wrong.

Nevertheless I am happy to add you as collaborator for slush on npm - just let me know your username there ;-)

**Comment by 4ndr01d3** [comment]
Hey guys, 
Thanks for replying, I share similar concerns about putting the build directory up there, however it was the easiest way that I came up to be able to keep using ```parcelify``` in the slush  to create the ```bundle.css```, while keeping the sniper happy about getting it from the ```build``` dir. Otherwise the user needs to manually point the ```sniper/css``` element in the ```package.json```. remember this also affects codepen, etc.
If you guys have other ideas for this, please let me know.
my username in npm is 4ndr01d3
Cheers,
Gustavo.

**Comment by greenify** [comment]
> If you guys have other ideas for this, please let me know.

Well finally agree on web components (biojs3), drop the entire browserify dependency and be similar to Polymer.

> npm owner add 4ndr01d3 slush-biojs

+ 4ndr01d3 (slush-biojs)

You should be ready to go.

**Comment by 4ndr01d3** [comment]
Thanks Seb.
I'm not saying no to the web components, but I think we need to have a solution for the current workflow, while the developments on the future BioJS get momentum.
